src/cms/templates/pages/page_tree_node.html

```diff
@@ -112,5 +112,5 @@
         </a>
         {% endif %}
         
-    </td>
+    </td>                      
```

Comment:
```suggestion
    </td>
```
Do not add unnecessary blanks